src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -53,6 +53,9 @@ public double getPieceOffestAngleX() {
   public double getPieceOffestAngleY() {
     return ty.getDouble(0);
   }
+  public double getTv(){
```

Comment:
A better name would be `isPieceSeen`. This one sounds like it porvides you with an appliance.

Reply:
done